src/selectors/characterSelectors.js

```diff
@@ -0,0 +1 @@
+export const characterSelectors = (state) => state.characters.character
```

Comment:
Rename to `character` since that is the piece of data you are selecting from the reducer